eZ/Publish/API/Repository/Tests/SearchServiceTranslationLanguageFallbackTest.php

```diff
@@ -1675,14 +1667,22 @@ public function testFindLocationsMultiple(
                 $data[$contentNo]['locations'][1]->id,
                 $location->id
             );
-            $this->assertEquals(
-                $this->getIndexName($indexMap),
-                $searchResult->searchHits[$realIndex]->index
-            );
+            $this->assertIndexName($indexMap, $searchResult->searchHits[$realIndex]);
             $this->assertEquals(
                 $translationLanguageCode,
                 $searchResult->searchHits[$realIndex]->matchedTranslation
             );
         }
     }
+
+    private function assertIndexName(array $indexMap, SearchHit $searchHit)
```

Comment:
```suggestion
    private function assertIndexName(array $indexMap, SearchHit $searchHit): void
```

Reply:
Fixed in https://github.com/ezsystems/ezpublish-kernel/pull/2928/commits/51a623da2f8169860197fea1443e4836ab4ae247